src/utils/SemaphoreTreeDepthValidator.sol

```diff
@@ -9,7 +9,7 @@ library SemaphoreTreeDepthValidator {
     ///
     /// @param treeDepth The tree depth to validate.
     /// @return supportedDepth Returns `true` if `treeDepth` is between 16 and 32 - depths supported by the Semaphore
-    function validate(uint8 treeDepth) public pure returns (bool supportedDepth) {
+    function validate(uint8 treeDepth) internal pure returns (bool supportedDepth) {
```

Comment:
This is a `library`, so you need to update the deployment script to account for this and deploy it properly.